sample/src/SampleAppRedirectOnLaunch.js

```diff
@@ -71,7 +71,7 @@ class SampleAppRedirectOnLaunch extends React.Component {
   clearRedirectInterval() {
```

Comment:
Do we need to explain this in the sample readme?

Reply:
I don't think so? it's not a functionality of the component, it just exists to do a good job of demonstrating the component